devicetypes/smartthings/aeotec-wallmote.src/aeotec-wallmote.groovy

```diff
@@ -181,8 +184,14 @@ private getButtonAttributesMap() {
             0: "pushed",
             2: "held",
             3: "double"
-    ]}
-    else {[
+    ]} else if (isWallMote7()) {[
+            0: "pushed",
+            2: "held",
+            3: "double",
+            4: "pushed_3x",
+            5: "pushed_4x",
+			6: "pushed_5x"
```

Comment:
Please correct spacing here (replace those tabs with spaces, 1 tab  = 4 spaces)

Reply:
Got it, this commit should fix the spacing on Line 193